packages/react/src/primitives/Button/Button.tsx

```diff
@@ -24,14 +28,30 @@ const ButtonPrimitive: Primitive<ButtonProps, 'button'> = (
     size,
     type = 'button',
     variation,
+    colorTheme,
     ...rest
   },
   ref
 ) => {
+  // Test if variation supports color themes.
+  const supportsColorThemes = supportedVariations.includes(variation);
+
+  // Use the variation to construct our color theme modifier classes; in the
+  // case of our 'default' variation, use 'outlined'
+  const variationColorThemeModifier = variation ?? 'outlined';
+
   const componentClasses = classNames(
     ComponentClassNames.Button,
     ComponentClassNames.FieldGroupControl,
     classNameModifier(ComponentClassNames.Button, variation),
+    // Check if variation supports colorThemes before applying
+    // colorTheme modifying class.
+    supportsColorThemes &&
+      colorTheme &&
+      classNameModifier(
+        ComponentClassNames.Button,
+        `${variationColorThemeModifier}--${colorTheme}`
+      ),
```

Comment:
Does `classNames` filter out `false` parameters?

Reply:
I think the above change^ addresses this (pulling out `classNameModifier` as a flag) and using 
```
classNameModifier(ComponentClassNames.Button, colorThemeModifier),
``` 
here instead